saleor/tests/e2e/channel/utils.py

```diff
@@ -34,6 +34,7 @@ def create_channel(
     currency="USD",
     country="US",
     is_active=True,
+    allowUnpaidOrders=False,
```

Comment:
Just to keep a consistency, I think you should have `allow_unpaid_orders`. (see format of the rest of arguments)

Reply:
I don't see usage of the new argument. Is it expected?

Reply:
@korycins https://github.com/saleor/saleor/pull/13551/commits/df70a2b98275bad73aab994935dc3d6689deb335

Reply:
Yeah, but you don't do anything with the `allowUnpaidOrders` that you received in the `create_channel` function. 
Additionally IMO, `allowUnpaidOrders` should be renamed to `allow_unpaid_orders` (I mean the name of argument for function)

Reply:
https://github.com/saleor/saleor/pull/13551/commits/20c95a2d68d1afc9874372ff5971a08ba0f8d519